docs/troubleshoot/app-framework/zss-error-codes.md

```diff
@@ -193,15 +193,29 @@ The following error message codes may appear on ZSS log. Use the following messa
 
 ### ZWES1601I
 
-  Server is ready to accept JWT `with`(or `without`) fallback to legacy tokens
+ZSS server is ready to accept JWT with (or without) fallback to legacy tokens.
```

Comment:
You cannot made edits to this line.
This is the line the log prints out. If you edit it, it would be incorrect.

`ZWES1601I Server is ready to accept JWT with fallback to legacy tokens`